### DISCUSSION TITLE
Different accent colour specifically for when listening to podcasts

### DISCUSSION THREAD

**Discussion by SmartWater2021** [ciderapp/Cider#1153] [Added Features]
I think having the option to choose a different accent colour for when you have the 'podcasts' tab open would be cool.
I.e. When listening to music, the accent colour would be the normal Cider red colour, but when listening to a podcast it'd change to be the purple of apple podcasts.

**Comment by Amaru8** [comment]
**Hi @SmartWater2021, I have some great news!** This feature is now implemented in the latest development version of **Cider**, and will be included in the next Microsoft Store update.
